src/pages/workspaces/workspace/analysis/ContextBar.js

```diff
@@ -201,11 +196,11 @@ export const ContextBar = ({
           tooltipSide: 'left',
           disabled: !isTerminalEnabled,
           href: terminalLaunchLink,
-          onClick: () => {
-            Ajax().Metrics.captureEvent(Events.analysisLaunch,
+          onClick: async () => {
+            await Ajax().Metrics.captureEvent(Events.analysisLaunch,
```

Comment:
I don't think you technically need to await the call to captureEvent because it is OK to continue on with the rest of the code before captureEvent completes. But there's also not harm in awaiting it.